packages/next-admin/src/components/common/Checkbox.tsx

```diff
@@ -15,7 +15,7 @@ export const Checkbox = ({ indeterminate, ...props }: Props) => {
     <input
       type="checkbox"
       ref={ref}
-      className="h-4 w-4 rounded border-gray-300 text-nextadmin-primary-600 focus:ring-nextadmin-primary-600 cursor-pointer"
+      className="h-4 w-4 rounded border-gray-300 text-indigo-600 focus:ring-indigo-600"
```

Comment:
Why don't we keep using the themable color ?

Reply:
typo! fixed